.github/workflows/web_tests.yml

```diff
@@ -6,16 +6,19 @@ permissions:
 jobs:
   web_tests:
     runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
This looks wrong - the matrix.os is not defined but above it is hard-coded to ubuntu

Reply:
Ah, right. Good catch